Makefile.am

```diff
@@ -57,6 +57,18 @@ shellcheck:
 			$$(find cmd/zpool/zpool.d/* -executable); \
 	fi
 
+mancheck:
+	@if type mandoc > /dev/null 2>&1; then \
+		for file in zfs zpool zdb zgenhostid; do \
+			mandoc -Tlint ${top_srcdir}/man/man8/$$file.8; \
+		done \
+	else \
+		type mandoc; \
+		sudo apt-get update ; sudo apt-get install -y mandoc; \
```

Comment:
I'm assuming this is temporary? I can get buildbot to install mandoc for you.

Reply:
Yes, it is temporary, i was only testing why it isnt't already installed ... Brian updated to buildslaves to Ubuntu Zesty (17) in https://github.com/zfsonlinux/zfs-buildbot/commit/aa7437e7e71f7694efbacaa59e8dfc8c2f15970e but somehow it seems it's still running Xenial (16) which doesn't ship `mandoc`.

Reply:
Give me one second. I'll look into that.

Reply:
Found the problem. Will get it fixed. Thanks for catching that.

Reply:
Thanks @dinatale2.  We'll see about updating the buildbot when there's a lull in testing.